Bottom: f6737619d6db54c0d8997c500925ce32946af3fe
Top:    0dd487508ba7b391db7cea3fd2ecbf63275e5b9f
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2016-04-29 01:44:06 -0500

Add plugin.info.txt


---

diff --git a/plugin.info.txt b/plugin.info.txt
new file mode 100644
index 0000000..d9a69bb
--- /dev/null
+++ b/plugin.info.txt
@@ -0,0 +1,7 @@
+base   tablecalc
+author Christopher Voltz
+email  cjunk@voltz.ws
+date   2016-04-29
+name   Table Calculations Plugin
+desc   Adds limited spreadsheet functionality to tables
+url    https://github.com/cvoltz/tablecalc.git
